tools/update.py

```diff
@@ -742,23 +731,112 @@ def _UninstallPackagesWindows(self, package_versions):
 
     return True
 
-  def UpdatePackages(self, package_names):
+  def UpdatePackages(self, projects_file, user_defined_project_names):
     """Updates packages.
 
     Args:
-      package_names (list[str]): package names that should be updated
-          if an update is available. An empty list represents all available
-          packages.
+      projects_file (str): path to the projects.ini configuration file.
+      user_defined_project_names (list[str]): user specified names or project
```

Comment:
I think this is meant to be "names of projects".

Reply:
done